pages/export.html

```diff
@@ -11,6 +11,7 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
         <link rel="stylesheet" href="../styles/theme.css">
         <script src="../scripts/export.js"></script>
+        <script src="../scripts/errors.js"></script>

```

Comment:
It (apparently) doesn't matter, since I've tested it and it's worked just fine. Odd for sure!